App/ListJS.js

```diff
@@ -0,0 +1,140 @@
+var jsonObj = {
+    playlist:
+           {
+            picture1: "../core/assets/massive-pop-hits.png",
+            title1: "text1",
+            description:"text2",
+            songs:
+            [
+                "value1",
+                "value2",
+                "value3",
+                "value4",
+                "value5"
+            ],
+            songPict:["../core/assets/beats-like-birds.png",
+            "../core/assets/chill-vibes.png",
+            "../core/assets/indie-rocker.png",
+            "../core/assets/random-playlist.png",
+            "../core/assets/beats-like-birds.png"]
+
+
+        }
+}
+var jsonUserName={
+      name:"username",
+      password:"password"
+}
+
+var playListFromJS= JSON.parse(playlistsJSON);
+
+
+document.addEventListener("DOMContentLoaded", function(event) {
+
+    var path = jsonObj.playlist.picture1;
+
+    document.getElementById('playL1').style.backgroundImage='url("'+path+'")';
+    var a=  Array.prototype.slice.call(jsonObj);
+
+  });
+// playlist class
+function Playlist(data) {
```

Comment:
Declare the full data model for Playlist and Song  and make sure you add getters and setters for them.